### PR TITLE
Known issues (#6)

### PR DESCRIPTION
**Pull request by fr1t2**
* Adding Ledger U2F Timeout issue

### CODE REVIEW

File: _docs/developers/knownIssues.md [hunk 1]
```diff
@@ -7,10 +7,41 @@ tags: developers
 
 There are a few known issues we have come across due to the software upstream of QRL. While we strive to overcome programmatically any issues, sometimes there are situations where limitations exist outside of the QRL scope. This document is intended to clear up any issues seen when installing and running The QRL software. 
 
-It should be made clear that these are not "Bugs" and there is very little we can do to mitigate these limitations. We strive to work through and provide secure, audited code in any of the official channels we operate.
+It should be made clear that these are not "Bugs" and there is very little we can do to mitigate these limitations. We strive to work through and provide secure, audited code in all of the official channels we operate.
```
Comment (jplomas) [inline _docs/developers/knownIssues.md]: "These are not bugs" where the next section talks about "There are a few small bugs".  This introduction should discuss that our project sits within an evolving ecosystem and as such there are likely to be issues that arise: some within our control and some without.  Individual components of our open source work have issues tracked on Github, and those which are important enough to need a wider audience than the developer community are documented here.

File: _docs/developers/knownIssues.md [hunk 2]
```diff
@@ -7,10 +7,41 @@ tags: developers
...
+
+**Mitigation -**
+
+Previously, we’ve recommended using another browser or playing browser bingo, switching to a browser that will still function with the U2F requirements of the Ledger. 
+
+The new current recommended action is to use our desktop wallets, which can be found on the [front page of our website](https://theqrl.org/) and our [github releases page](https://github.com/theQRL/qrl-wallet/releases/latest).
+
+You can read more about the development of U2F tunnel transport for Ledger devices in their blog [Windows 10 Update: Sunsetting U2F tunnel transport for Ledger devices](https://www.ledger.com/2019/05/17/windows-10-update-sunsetting-u2f-tunnel-transport-for-ledger-devices/).
+
+> If you have attempted to send a TX with your ledger device connected to a Chrome, Brave, Opera, or Firefox browser and are showing and erroneous OTS key, you can reset the ledger device OTS key count. See the tools section of your opened wallet using the desktop application. **Make Sure you have not used any keys successfully prior to resetting the ledger**
```
Comment (jplomas) [inline _docs/developers/knownIssues.md]: Can also mention that a rewrite of our Ledger javascript library is underway to take advantages of recent improvements in Ledger's software development kit.

File: _docs/developers/knownIssues.md (hunk 1 above)
Comment (fr1t2) [reply _docs/developers/knownIssues.md]: Updated to align with above.

File: _docs/developers/knownIssues.md (hunk 2 above)
Comment (fr1t2) [reply _docs/developers/knownIssues.md]: Done

**Pull request merged**